railties/lib/rails/commands/destroy/destroy_command.rb

```diff
@@ -14,14 +15,21 @@ def help
         end
       end
 
+      def delete_css_file_generate_with_scaffold
+        path = Rails.root.join('app', 'assets', 'stylesheets', 'scaffolds.scss')
+        FileUtils.remove_file(path,force=true)
+        puts " "*6+"\e[31mremove\e[0m"+" "*4  + path.to_s.split("/").reverse.slice(0,4).reverse.join("/")
```

Comment:
This feels like we're doing more work here than we need to, a couple of suggestions:

* Make sure there isn't a better place for this helper function
* Separate the `FileUtils.remove_file` logic from the `puts <massive-string-expression>`
* Add tests for `<massive-string-expression>`
* Add a railtie test to cover this usage

I think we'll need to see those to feel confident about adding this code to Rails.